labs/lab08-airgap-lite/README.md

```diff
@@ -1,111 +1,106 @@
 
 ***
-## Airgap Workflow Overview
+## Airgap workflow overview
 
-First, we'll push a release -- in the background, Replicated's Air Gap builder will prepare an Air Gap bundle.
+First, we'll push a release and Replicated's Air Gap builder will prepare an Air Gap bundle.
 
 ![airgap-slide-1](img/airgap-slide-1.png)
 
-Next, we'll collect a license file, a download link, and a public kURL bundle.
+Next, we'll collect a license file, a download link, and a public Kubernetes installer bundle.
 
 ![airgap-slide-2](img/airgap-slide-2.png)
 
 From there, we'll move all three artifacts into the datacenter via a jump box.
 
 ![airgap-slide-3](img/airgap-slide-3.png)
 
-The above diagram shows a three node cluster, but we'll use only a single node.
-While the KOTS bundle will be moved onto the server via SCP as in the diagram,
+The above diagram shows a three node cluster, but this lab will only use a single node cluster.
+The app manager bundle will be moved onto the server via SCP, while
 the app bundle and license file will be uploaded via a browser UI through an SSH tunnel.
 
 ***
-## Getting Started
+## Getting started
 
 
-You should have received an invite to log into https://vendor.replicated.com -- you'll want to accept this invite and set your password.
+You will receive an invite to log into https://vendor.replicated.com, accept this invite and set your password.
 
-From the Settings page, copy the Application Slug. You'll need that later to set the `REPLICATED_APP`.
+From the Settings page, copy the Application Slug. You'll need this slug in a later step to set the `REPLICATED_APP`.
 
 ![kots-app-slug](img/application-slug.png)
 
 ***
-## Instance Overview
+## Instance overview
 
-You will have received the IP of a jump box and the name of an Air Gap server.
-For example, you may have received:
+You will receive the IP of a jump box and the name of an Air Gap server.
+
+For example:
 
 ```text
 dx411-dex-lab08-airgap-lite-jump = 104.155.131.205
 dx411-dex-lab08-airgap-lite
 ```
 
 In general, the name of the private server will be the same as the jump box, with the characters `-jump` removed from the suffix.
-Put another way, you could construct both instance names programatically as
+
+For example, you can construct both instance names programmatically as:
 
 ```shell
 ${REPLICATED_APP}-lab08-airgap-lite-jump
 ```
 
-and
 
 ```shell
 ${REPLICATED_APP}-lab08-airgap-lite
 ```
 
-### Connecting
-
-First set your application slug, the public IP of your jump box and your first name:
+### Connecting to the Air Gap server
 
+1. Set your application slug, the public IP of your jump box and your first name:
 ```shell
 export JUMP_BOX_IP=...
 export REPLICATED_APP=... # your app slug
 export FIRST_NAME=... # your first name (lower case)
 ```
 
-Next, you can SSH into the Air Gap server using the following command:
-
+2. SSH into the Air Gap server using the following command:
```

Comment:
```suggestion
1. SSH into the Air Gap server using the following command:
```
Can we let markdown figure out the numbering. If you keep all of them at `1.`, markdown rendering will figure it out.

Reply:
Same as above. I suggest we indent the below code block.